src/Octokit.Webhooks.AspNetCore/GitHubWebhookExtensions.cs

```diff
@@ -36,7 +36,8 @@ public static class GitHubWebhookExtensions
                 try
                 {
                     var service = context.RequestServices.GetRequiredService<WebhookEventProcessor>();
-                    service.ProcessWebhook(context.Request.Headers, body);
+                    await service.ProcessWebhookAsync(context.Request.Headers, body)
```

Comment:
Should we pass through the `CancellationToken` from `context.RequestAborted`? Genuinely not sure 🤔

Reply:
If you are wanting to pass a cancellation token the whole way down then I think it would make sense to use the `httpContext.RequestAborted`  property. 

Happy to do this if you want, however I can't really think of a good use case for it. My preference would be to keep the interface lean and add it when there is a need. What are your thoughts?

Reply:
Yeah, makes sense. I just know in general it's a best practice to pass the cancellation token down the stack for cooperative cancellation.